packages/markup-include/src/components/EditorPanel.tsx

```diff
@@ -0,0 +1,526 @@
+          <div className='d-flex justify-content-between'>
+            <button onClick={handleVariableDoneClick} disabled={addConditionDisabled}>
+              <Icon display='caretUp' />
+            </button>
+            <button
+              className='btn btn-danger btn-sm mt-0 ml-2'
+              onClick={event => {
+                event.preventDefault()
+                deleteVariable(variableIndex)
+              }}
+            >
+              Delete
+            </button>
+          </div>
+          <label className='d-block'>
+            <span className='edit-label column-heading'>Variable Name:</span>
+            <input className={`variable-${variableIndex} ml-1`} type='text' value={variableName} onChange={e => setVariableName(e.target.value)} />
+          </label>
+          <div className='pt-2'>
+            <label className='d-block'>
+              <span className='edit-label column-heading'>Column:</span>
+              <select className={`variable-${variableIndex} ml-1`} onChange={e => handleVariableColumnChange(e.target.value)} value={selectedColumn} disabled={columnSelectDisabled}>
+                <option value=''>Select</option>
+                {columnNames.map(columnName => (
+                  <option key={columnName} value={columnName}>
+                    {columnName}
+                  </option>
+                ))}
+              </select>
+            </label>
+          </div>
+          <label className='d-block'>
+            <span className='edit-label column-heading'>Conditions:</span>
+            {conditionsList.map((condition, index) => {
+              return (
+                <div className='condition-section mt-2'>
+                  <Condition
+                    key={variableName + '-condition-' + index}
+                    conditionControls={openConditionControls}
+                    conditionLookup={conditionLookup}
+                    conditionSettings={condition}
+                    number={index}
+                    removeCondition={removeCondition}
+                    selectedColumn={selectedColumn}
+                    updateConditionsList={updateConditionsList}
+                  />
+                </div>
+              )
+            })}
+          </label>
+          <div className='mt-1'>
+            <button onClick={handleAddConditionClick} disabled={addConditionDisabled}>
+              Add Condition
+            </button>
+            <button className='ml-2' onClick={handleVariableDoneClick} disabled={addConditionDisabled}>
+              Done
+            </button>
+          </div>
+        </fieldset>
+      )}
+    </>
+  )
+}
+
+type CheckBoxProps = {
+  fieldName: string
+  label: string
+  section?: string
+  subsection?: string
+  tooltip?: string
+  updateField: Function
+  value: boolean
+}
+
+const CheckBox: React.FC<CheckBoxProps> = memo(({ fieldName, label, subsection = null, tooltip, section = null, updateField, value, ...attributes }) => (
```

Comment:
can we refactor and use this checkbox? \packages\core\components\inputs\InputCheckbox.jsx